### ISSUE TITLE
Blockchain

### ISSUE THREAD

**Issue by EsteveAguilera** [InformaticsEmporda/Xerrades#20]
### Proposta

- [x] Proposo que algú faci una xerrada
- [ ] Vull fer una xerrada
- [ ] Conec algú que la pot fer

#### Breu descripció
Com fer-nos rics o arruinar-nos de manera digital i no morir en l'intent!

**Comment by alemany** [comment]
Ei, aqui hi ha dues possibles xerrades Cryptomonedes i l'altre conceptes de Blockchain técnics.

**Comment by EsteveAguilera** [comment]
Ok @alemany, així doncs potser millor separar-ho en dues:

- Una primera xerrada de Cryptomonedes, tipus, funcionament i conceptes bàsics
- Una segona xerrada de la part més técnica del blockchain

Com ho veus? T'animes a donar-ne alguna? Les dues? 😅

**Comment by alemany** [comment]
si, una si... la que volgueu.. posem a votació ;)

- [ ]  Cryptomonedes 
- [ ]  Blockchain técnic

**Comment by EsteveAguilera** [comment]
Yay! Jo faig 👍 per a la de **Cryptomonedes**!

**Comment by eymbert** [comment]
👍🏻 a totes dues! 😂
